File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ComboBox/Features/CasingPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.ComboBox.Features
{
    public partial class CasingPage : UserControl
    {
        public CasingPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles the SelectedIndexChanged event of the objCasingComboBox control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjCasingComboBox_SelectedIndexChanged(object sender, EventArgs e)
        {
            //Sets selected type of CharacterCasing to combo

            mobjTestComboBox.CharacterCasing = mobjCasingComboBox.SelectedIndex == 0 ? CharacterCasing.Normal : (mobjCasingComboBox.SelectedIndex == 1 ? CharacterCasing.Upper : CharacterCasing.Lower);
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ComboBox/Features/CasingPage.vb
Namespace CompanionKit.Controls.ComboBox.Features

	Public Class CasingPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handles the SelectedIndexChanged event of the objCasingComboBox control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        Private Sub mobjCasingComboBox_SelectedIndexChanged(sender As Object, e As EventArgs) Handles mobjCasingComboBox.SelectedIndexChanged
            'Sets selected type of CharacterCasing to combo
            If (mobjCasingComboBox.SelectedIndex = 0) Then
                mobjTestComboBox.CharacterCasing = CharacterCasing.Normal
            Else
                If (mobjCasingComboBox.SelectedIndex = 1) Then
                    mobjTestComboBox.CharacterCasing = CharacterCasing.Upper
                Else
                    mobjTestComboBox.CharacterCasing = CharacterCasing.Lower
                End If
            End If
        End Sub
    End Class

End Namespace